sympy/solvers/tests/test_inequalities.py

```diff
@@ -388,12 +388,9 @@ def test_issue_10198():
 
 
 def test_issue_10047():
-    # issue 10047: this must remain an inequality, not True, since if x
-    # is not real the inequality is invalid
-    # assert solve(sin(x) < 2) == (x <= oo)
-
     # with PR 16956, (x <= oo) autoevaluates when x is extended_real
     assert solve(sin(x) < 2) == True
```

Comment:
This really shouldn't be evaluating as True for reasons given in the comment that you deleted. `x` was not declared in this file to be `extended_real` so this shouldn't have evaluated. If it does (and did at the time of the comment) it should be opened as an issue. Please leave the comment and put a "# XXX need to fix" instead of deleting the comment.

Reply:
This is where it was changed:
https://github.com/sympy/sympy/pull/16956/files#r291032047

Reply:
As I said above, this is beyond the scope of this PR:

> I ended up changing it back to the original test (although I left the second test that I added > there), so that it passes with the current implementation of inequality solvers. Only because > that issue is beyond the scope of this PR. See issue #17840

However, I could restore the test and mark it as expected to fail, perhaps that would be better

Reply:
I think if you restore that comment and perhaps add to it if there is anything else useful to add then this can be merged

Reply:
Yeah, okay. Works for me